doc/cascadia/profiles.schema.json

```diff
@@ -0,0 +1,449 @@
+{
+  "$id": "https://github.com/microsoft/terminal/terminal.profile.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Microsoft Terminal Profile Settings",
+  "definitions": {
+    "Color": {
+      "default": "#",
```

Comment:
What does `#` mean as a default?

Reply:
When editing the profiles.json, it'll auto-fill the "#" for the color code